psi4/driver/procrouting/solvent/pol_embed.py

```diff
@@ -42,8 +44,20 @@ def get_pe_options():
     rmin = core.get_option('PE', 'BORDER_RMIN')
     if core.get_option('PE', 'BORDER_RMIN_UNIT').upper() == "AA":
         rmin *= 1.0 / constants.bohr2angstroms
+
+    # potfile option can be filename or contents
+    potfile_keyword = core.get_option('PE', 'POTFILE')
+    if len(potfile_keyword) > 300:
```

Comment:
300?

Reply:
it was an attempt to differentiate between a long file path and file contents. I'm switching to testing on `@COORDINATES`.